### PR TITLE
Changes related to Securonix release 2.0.13

### PR DESCRIPTION
**Pull request by scnx-knathani**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Updated ***securonix-get-incident*** and ***securonix-list-incidents*** commands to include the Spotter query in the HR and context by upgrading to the latest version of the API.
- Updated the ***securonix-list-incidents*** command to include the Incident ID field in the HR column.
- Updated the ***securonix-list-violation-data*** command, where the **query** argument now supports the auto inclusion of the "index" with a default value as "violation" if not provided.
- Updated the Docker image to: *demisto/python3:3.10.13.78960*.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Securonix/ReleaseNotes/2_0_13.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+#### Integrations
+
+##### Securonix
+
+- Updated the ***securonix-get-incident*** and ***securonix-list-incidents*** commands to include the Spotter query in the HR and context by upgrading to the latest version of the API.
```
Comment (ShirleyDenkberg) [inline Packs/Securonix/ReleaseNotes/2_0_13.md:6]:
```suggestion
- Updated the following commands to include the Spotter query in the HR and context by upgrading to the latest version of the API.
   - ***securonix-get-incident*** 
   - ***securonix-list-incidents***
```

What does HR stand for?

File: Packs/Securonix/ReleaseNotes/2_0_13.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Updated the ***securonix-list-incidents*** command to include the Incident ID field in the HR column.
+- Updated the ***securonix-list-violation-data*** command, where the **query** argument now supports the auto inclusion of the "index" with a default value as "violation" if not provided.
```
Comment (ShirleyDenkberg) [inline Packs/Securonix/ReleaseNotes/2_0_13.md:8]:
```suggestion
- Updated the ***securonix-list-violation-data*** command, where the *query* argument now supports the auto inclusion of the "index" with a default value of "violation" if not provided.
```

File: Packs/Securonix/ReleaseNotes/2_0_13.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+#### Integrations
+
+##### Securonix
+
+- Updated the ***securonix-get-incident*** and ***securonix-list-incidents*** commands to include the Spotter query in the HR and context by upgrading to the latest version of the API.
+- Updated the ***securonix-list-incidents*** command to include the Incident ID field in the HR column.
```
Comment (ShirleyDenkberg) [inline Packs/Securonix/ReleaseNotes/2_0_13.md:7]: What does HR stand for?

File: Packs/Securonix/ReleaseNotes/2_0_13.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+#### Mappers
+
+##### Securonix Incident - Incoming Mapper
+
+- Updated the mapper for the ***Securonix Violation Spotter Query*** incident field.
+
+##### Securonix Threat - Incoming Mapper
+
+- Updated the mapper for the ***Securonix Violation Spotter Query*** incident field.
```
Comment (ShirleyDenkberg) [inline Packs/Securonix/ReleaseNotes/2_0_13.md:19]:
```suggestion
Updated the mapper for the *Securonix Violation Spotter Query* incident field.

##### Securonix Threat - Incoming Mapper

Updated the mapper for the *Securonix Violation Spotter Query* incident field.
```

**Comment by ShirleyDenkberg** [comment]
@melamedbn @MosheEichler Doc review completed.

File: Packs/Securonix/ReleaseNotes/2_0_13.md (hunk 1 above)
Comment (MosheEichler) [reply Packs/Securonix/ReleaseNotes/2_0_13.md:6]: Human Readable

**Comment by melamedbn** [comment]
Hi @scnx-knathani,

Can you please explain the changes made to the mappers?

Best regards,
Ben

**Comment by scnx-knathani** [comment]
> Hi @scnx-knathani,
> 
> Can you please explain the changes made to the mappers?
> 
> Best regards, Ben

Previously we removed the `index=violation` from the Spotter query using the transformers in the Mapper but from now onwards this will be handled in the command logic itself we'll show the full query to the end user without trimming the layout hence this change was made in the Mapper. Please let me know if you've further questions on this.

**Review by melamedbn — APPROVED** [review]

**Pull request merged**